modules/snooker/src/main/ts/ephox/snooker/api/TableDirection.ts

```diff
@@ -1,22 +1,3 @@
-import { Optional } from '@ephox/katamari';
-import { SugarElement } from '@ephox/sugar';
-import { BarPositions, ColInfo } from '../resize/BarPositions';
-import { ResizeDirection } from './ResizeDirection';
+import { width, height } from '../resize/BarPositions';
 
-export type TableDirection = BarPositions<ColInfo>;
-
-export const TableDirection = (directionAt: (e: SugarElement) => { isRtl: () => boolean }): TableDirection => {
-  const auto = (table: SugarElement) => directionAt(table).isRtl() ? ResizeDirection.rtl : ResizeDirection.ltr;
-
-  const delta = (amount: number, table: SugarElement) => auto(table).delta(amount, table);
-
-  const positions = (cols: Optional<SugarElement>[], table: SugarElement) => auto(table).positions(cols, table);
-
-  const edge = (cell: SugarElement) => auto(cell).edge(cell);
-
-  return {
-    delta,
-    edge,
-    positions
-  };
-};
+export { height, width };
```

Comment:
Am I missing something? How did the directions module end up exporting things called height and width?

Reply:
Because, as it turns out, BarPositions data is only ever used in height and width calculations. These names seemed more helpful. TableDirection should probably be renamed tbh, but that can be a later problem.